docs/src/release-notes-js.md

```diff
@@ -8,6 +8,48 @@ import LiteYouTube from '@site/src/components/LiteYouTube';
 
 ## Version 1.49
 
+### Aria snapshots
+
+New assertion [`method: LocatorAssertions.toMatchAriaSnapshot`] provides a way to verify page structure by comparing it with an expected YAML representation of the accessibility tree.
+
+```js
+await page.goto('https://playwright.dev');
+await expect(page.locator('body')).toMatchAriaSnapshot(`
+  - banner:
+    - heading /Playwright enables reliable/ [level=1]
+    - link "Get started"
+    - link "Star microsoft/playwright on GitHub"
+  - main:
+    - img "Browsers (Chromium, Firefox, WebKit)"
+    - heading "Any browser • Any platform • One API"
+`);
+```
+
+You can generate this assertion with [Test Generator](./codegen) and update the expected snapshot with `--update-snapshots` command line flag.
+
+Learn more in the [aria snapshots guide](./aria-snapshots).
+
+### Test runner
+
+- New option [`property: TestConfig.tsconfig`] allows to specify a single `tsconfig` to be used for all tests.
+- New method [`method: Test.fail.only`] to focus on a failing test.
+- Options [`property: TestConfig.globalSetup`] and [`property: TestConfig.globalTeardown`] now support multiple setups/teardowns.
+- New value `'on-first-failure'` for [`property: TestOptions.screenshot`].
+- Added "previous" and "next" buttons to the HTML report to quickly switch between test cases.
+- New properties [`property: TestInfoError.cause`] and [`property: TestError.cause`] mirroring [`Error.cause`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause).
+
+### Miscellaneous
+
+- `<canvas>` elements inside a snapshot now draw a preview.
```

Comment:
Lets wait with landing until https://github.com/microsoft/playwright/issues/33562#issuecomment-2471874482 is resolved?

Reply:
Agree!

Reply:
We'll update if needed.